openapi/newsapi/.gitignore

```diff
@@ -0,0 +1 @@
+target
```

Comment:
We had better add other files or directories that should be ignored. like configs and others.

Reply:
I guess it is autogenerated when you do the `bal new <name> -t lib` command. The root `.gitignore` has the necessary configs.